guides/source/rails_on_rack.md

```diff
@@ -233,6 +239,10 @@ Much of Action Controller's functionality is implemented as Middlewares. The fol
 
 * Used for thread safe code reloading during development.
 
+**`ActionDispatch::ServerTiming`**
+
+* Sets a Server-Timing header, containing one or more metrics and descriptions for the given request-response cycle.
```

Comment:
Again, I realize this is [exactly what MDN says](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Server-Timing), but I feel like we can tailor it a bit for our case:

```suggestion
* Sets a [`Server-Timing`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Server-Timing) header containing performance metrics for the request.
```

Reply:
Done